tools/docker/boardfarm-ci/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM python:3.8-slim-buster
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    apt-transport-https \
+    ca-certificates curl \
+    curl \
+    gcc \
+    git \
+    gnupg \
+    gnupg-agent \
+    libsnmp-dev \
+    software-properties-common \
+    wireshark-common \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY requirements.txt /app/requirements.txt
+
+WORKDIR app
+RUN pip3 install -r requirements.txt
+
+# TODO: what needs this?
+#RUN pip3 install jsonschema distro
+
+
+RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
+RUN add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable"
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    docker-ce \
+    docker-ce-cli \
+    containerd.io \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+RUN chmod 755 /usr/local/bin/docker-compose
```

Comment:
For my info: why isn't all this simply taken from Debian/Ubuntu repos? Are we using such advanced features?

Reply:
Good question, I took the dockerfile from https://github.com/prplfoundation/prplMesh/pull/1483 and tried to make only the changes I needed.
Maybe @odkq needs a more recent version than the base image provides?

Reply:
We could, debian stable is 1.21 and I don't think we are going to use later development, but as documentation is not tagged (always the latest) with this we have the mechanism in place for future upgrades (just change the 1.26.0 in the url and test)